File: src/main/java/com/example/healthylifestyleapp/PhysiquesDetailsActivity.java
package com.example.healthylifestyleapp;

/*
import android.support.v7.app.AppCompatActivity;
*/
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.AppCompatButton;

public class PhysiquesDetailsActivity extends AppCompatActivity implements View.OnClickListener {
    AppCompatButton AppCompatButtonFinish;
    Context context=this;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_physiques_details);
        AppCompatButtonFinish=findViewById(R.id.AppCompatButtonFinish);
        AppCompatButtonFinish.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId())
        {
            case R.id.AppCompatButtonFinish:
                //Luanching the new activity
                Intent newIntent= new Intent(this, ConfirmDetailsActivity.class);
                startActivity(newIntent);
        }
    }
}

File: app/src/main/java/com/example/healthylifestyleapp/ui/activities/PhysiquesDetailsActivity.kt
package com.example.healthylifestyleapp.ui.activities

/*
import android.support.v7.app.AppCompatActivity;
*/

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.healthylifestyleapp.R
import kotlinx.android.synthetic.main.activity_physiques_details.*

class PhysiquesDetailsActivity : AppCompatActivity(), View.OnClickListener {
    internal var context: Context = this

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_physiques_details)
        AppCompatButtonFinish.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.AppCompatButtonFinish -> {
                //Luanching the new activity
                val newIntent = Intent(this, ConfirmDetailsActivity::class.java)
                startActivity(newIntent)
            }
        }
    }
}
